tests/rules/unqualified-attributes.js

```diff
@@ -13,6 +13,16 @@
             Assert.areEqual("Unqualified attribute selectors are known to be slow.", result.messages[0].message);
         },
 
+        "A selector that contains a class is considered qualified": function(){
```

Comment:
To match the bit below, I'd change this to:

Using an attribute selector qualified by a class should not result in a warning
I'd also move it to be nearer to line 38 test since they go together.